src/types.jl

```diff
@@ -98,6 +98,16 @@ end
 ###
 ### Term interface
 ###
+
+"""
+  symtype(x)
+
+Returns the symbolic type of `x`. By default this is just `typeof(x)`.
+Define this for your symbolic types if you want `SymbolicUtils.simplify` to apply rules
+specific to numbers (such as commutativity of multiplication). Or such
+rules that may be implemented in the future.
+"""
+symtype(x) = typeof(x)
 symtype(x::Number) = typeof(x)
```

Comment:
We could just leave one method for `symtype`

Reply:
this isn't correct though? It should be the type parameter.

Reply:
Yes, it's the type parameter. I can group the definitions together.